### ISSUE TITLE
Adding directory support

### ISSUE THREAD

**Issue by rimazmk** [test-room-7/action-update-file#4]
It would be nice if changing all files in a directory by specifying the directory could be supported. Right now if I have `directory/` as my `file-path` I get this error: `EISDIR: illegal operation on a directory, read`.

**Comment by alexesprit** [comment]
Yes, directories aren't supported.

I would resolve the issue by adding support for globs:
```yml
- name: Update resources
  uses: test-room-7/action-update-file@v1
  with:
    # All files from `dir` directory
    file-path: dir/*
```

**Comment by alexesprit** [comment]
I'll update documentation and publish the new version with glob support.

**Comment by alexesprit** [comment]
@rimazk123 Hi! I've published v1.3.0 which supports glob patterns. You can use the `dir/*` pattern to include all files from the `dir` directory only, or the `dir/**/*` patterns to include all files from the `dir` directory, and from all subdirectories. This is very common syntax for glob patterns.

This way is more flexible and more straightforward, as there no directories in git.

Please let me know if you have any questions.
